spec/models/user_spec.rb

```diff
@@ -12,15 +15,21 @@
     expect(user).to_not be_valid
     expect(user.errors[:password_confirmation]).to_not be_blank
   end
-end
+
+  describe 'Confirm username' do
+    it 'should have a username' do
+      expect(user.username).to eq(username)
+    end
+  end
+
 
 RSpec.describe 'Confirm admin role' do
-  it 'is not an admin if the role is not admin' do
+  it 'admin? returns false if user is not an admin' do
     user = FactoryGirl.create(:user)
```

Comment:
Your user factory's default mode is 'member', so you should be able to reuse the user factory created at the top of this describe block for this test